cmd/eksctl-anywhere/cmd/vspheresetupuser.go

```diff
@@ -0,0 +1,86 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+
+	"github.com/aws/eks-anywhere/pkg/dependencies"
+	"github.com/aws/eks-anywhere/pkg/providers/vsphere/setupuser"
+)
+
+type vSphereSetupUserOptions struct {
+	fileName string
+	force    bool
+	password string
+}
+
+var vsuo = &vSphereSetupUserOptions{}
```

Comment:
Given this is package state (which I hope we remove across the board for commands in the future) do you think we could expand its name even if that means repeating the type? Its similar to my other comment about usage far from the site of declaration.